src/index.ts

```diff
@@ -273,12 +276,19 @@ async function handleKanbanMessage(msg: Action) {
  * the domain of the current board.
  */
 async function handleNewlyOpenedNote(newNoteId: string) {
+
   if (openBoard) {
     if (openBoard.configNoteId === newNoteId) return;
     if (await openBoard.isNoteIdOnBoard(newNoteId)) return;
     else {
```

Comment:
is this "else" usefull?

Reply:
Yes. Let's say you have opened Kanban A note , and then open another Kanban B note, the else code handle this condition and replace the the board to Kanban B. If no such code, you need to close the current board  and then open the Kanban note B manually.

Reply:
Remark: The reloadConfig() replace the "openBoard" variable. That is why it looks a bit weird.